submissions/pozzitive11/frogger-game/app.js

```diff
@@ -0,0 +1,117 @@
+const tileSize = {
+  width: 101,
+  height: 85,
+};
+
+const enemySpeed = {
+  first: 90,
+  second: 230,
+  third: 150,
+};
+
+const boardSize = {
+  top: 0,
+  right: tileSize.width * 5,
+  bottom: tileSize.height * 6,
+  left: 0,
+};
+
+class gameObjectRender {
```

Comment:
Think of class hierarchy as of bilogical taxonomy. Humans inherit features from apes, apes from mammals, mammals from vertebraes, and so on.
In this fictional game world bugs and player have a common fictional predecessor whose name could be saying something to the developers that may want to contribute to your game. May be a both bugs and player are instances of a character?